### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by MartinK1983**
First Steps my App "Free DogGo"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "next": "13.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^5.3.8"
+    "react-hook-form": "^7.43.5",
+    "styled-components": "^5.3.8",
+    "use-local-storage-state": "^18.2.0"
```
Comment (marcbendixen) [inline package.json:20]:
```suggestion
    "styled-components": "^5.3.8",
```
If you don't use certain packages at the moment, they shouldn't be part of a feature/PR

File: pages/index.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
 import styled from "styled-components";
-
 export default function Home() {
   return (
     <main>
       <Heading>🐋Capstone Whales Template🐋</Heading>
```
Comment (marcbendixen) [inline pages/index.js]:
```suggestion
```
I don't think you need this party anymore 😉

File: pages/index.js [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+      </StyledWelcome>
     </main>
   );
 }
 
 const Heading = styled.h1`
-text-align: center;
-`;
+text-align: center;`
+
```
Comment (marcbendixen) [inline pages/index.js]:
```suggestion
```
I don't think you need this party anymore 😉

**Review by marcbendixen — CHANGES_REQUESTED** [review]
Good job 🚀 
I made a few comments.

**Comment by Kheirah** [comment]
> First Steps my App "Free DogGo"

That's not saying much. What exactly did you implement? Please leave a link to the corresponding user story.
**Be more descriptive** in your PR next time.

**Pull request closed without merging**